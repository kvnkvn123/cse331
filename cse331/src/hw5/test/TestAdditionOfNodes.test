# Tests that a graph can be created, a single node added and
# a second node added. Then tests by adding the same node
# (should not be added) 

# Create a graph
CreateGraph graph1

# Add a node
AddNode graph1 n1

# Add another node
AddNode graph1 n2

# Add the same node as before
AddNode graph1 n2