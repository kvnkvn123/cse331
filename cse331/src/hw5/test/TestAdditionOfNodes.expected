# Tests that a graph can be created, a single node added and
# a second node added. Then tests by adding the same node
# (should not be added) 

# Create a graph
created graph graph1

# Add a node
added node n1 to graph1

# Add another node
added node n2 to graph1

# Add the same node as before
added node n2 to graph1